src/main/java/com/uooconnect/main/BrowserStart.java

```diff
@@ -0,0 +1,45 @@
+package com.uooconnect.main;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import project.utility.com.BrowserStartUtility;
+
+import java.io.IOException;
+
+public class BrowserStart {
+    static WebDriver driver;
+    static int sheet1count;
+    static BrowserStartUtility startUtility;
+
+    static {
+        try {
+            startUtility = new BrowserStartUtility();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
```

Comment:
@mallishwarigunda Framework classes not need to have main method.

Reply:
No business logic code is needed in main method for Framework classes.

Reply:
Resolved this.